features/mbedtls/importer/adjust-config.sh

```diff
@@ -37,38 +37,39 @@ add_code() {
 
 # add an #ifndef to include config-no-entropy.h when the target does not have
 # an entropy source we can use.
-add_code                                                                                  \
-    "#ifndef MBEDTLS_CONFIG_H\n"                                                          \
-    "\n"                                                                                  \
-    "#include \"platform\/inc\/platform_mbed.h\"\n"                                       \
-    "\n"                                                                                  \
-    "\/*\n"                                                                               \
-    " * Only use features that do not require an entropy source when\n"                   \
-    " * DEVICE_ENTROPY_SOURCE is not defined in mbed OS.\n"                               \
-    " *\/\n"                                                                              \
-    "#if !defined(MBEDTLS_ENTROPY_HARDWARE_ALT) && !defined(MBEDTLS_TEST_NULL_ENTROPY)\n" \
-    "#include \"mbedtls\/config-no-entropy.h\"\n"                                         \
-    "\n"                                                                                  \
-    "#if defined(MBEDTLS_USER_CONFIG_FILE)\n"                                             \
-    "#include MBEDTLS_USER_CONFIG_FILE\n"                                                 \
-    "#endif\n"                                                                            \
-    "\n"                                                                                  \
+add_code                                                                                          \
```

Comment:
I do not think there is such an error. I just added a one-byte space to align the end of line.

Reply:
I have checked on my machine, and the diff is because the alignment of the ending `\` has changed:
![image](https://user-images.githubusercontent.com/22030476/45073812-92b4ef80-b0e9-11e8-8c1e-76653e013ffc.png)
This is because the addition of `\\\\` at line 49 to add the `"    !defined(MBEDTLS_ENTROPY_NV_SEED)\n"` part in line 50, i believe

Reply:
@RonEld Thank you for comments, it is correct.